### ISSUE TITLE
Add hover label for icons and/or buttons

### ISSUE THREAD

**Issue by mhstoller** [proksh/atomize#12]
I'm not sure if this is already available, but it would be nice to have the ability to add a hover label to an icon or button. You can add the "title" attribute to a button and have the standard hover label/tool-tip display, but it doesn't match the aesthetic of the UI, and doesn't align with the element nicely.

```
                        <Button bg="white" title="test">
                            <Icon name="Logout" size="2rem" />
                        </Button>
```

This will display a label of "test" when hovering over the element, for example.

I'm not sure if what I'm asking is even possible, but just figured I would throw it out there.

**Comment by proksh** [comment]
This is the default for the browser, I have a tooltip on the roadmap. Hope that will solve your issue.

**Comment by mhstoller** [comment]
Yes, that sounds good. Should I leave this issue open or close it?

**Comment by proksh** [comment]
You can leave this open. :)

**Comment by bobbylemm** [comment]
Hello @proksh has this issue been fixed

**Comment by LucaMoro96** [comment]
@bobbylemm Unfortunately not!

**Comment by bobbylemm** [comment]
@LucaMoro96 can I work on this

**Comment by LucaMoro96** [comment]
Yes!

**Comment by bobbylemm** [comment]
Alright, great
